PWGHF/TableProducer/treeCreatorD0ToKPi.cxx

```diff
@@ -154,97 +158,124 @@ struct HfTreeCreatorD0ToKPi {
   {
   }
 
-  void process(aod::Collisions const& collisions,
-               aod::McCollisions const& mccollisions,
-               soa::Join<aod::HfCand2Prong, aod::HfCand2ProngMcRec, aod::HfSelD0> const& candidates,
-               soa::Join<aod::McParticles, aod::HfCand2ProngMcGen> const& particles,
-               aod::BigTracksPID const& tracks)
+  template <typename T>
+  void fillEvent(const T& collision, int8_t isEventReject, int8_t runNumber)
   {
+    rowCandidateFullEvents(
+      collision.bcId(),
+      collision.numContrib(),
+      collision.posX(),
+      collision.posY(),
+      collision.posZ(),
+      isEventReject,
+      runNumber);
+  }
+
+  template <typename T, typename U, typename V>
+  auto fillTable(const T& candidate, const U& prong0, const V& prong1, int CandFlag, int FunctionSelection, double FunctionInvMass, double FunctionCosThetaStar,
+                 double FunctionCt, double FunctionY, double FunctionE, int8_t FlagMc, int8_t Origin)
```

Comment:
The arguments are all variables so they should follow the same `camelCase` convention.